fixtures/ext-types/uniffi-one/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 # TODO: modify the crate name to test non-default names.
```

Comment:
I'd be inclined to delete this comment while you are here - I've no idea what it means, and I suspect I added it :)